[cms] politeiawww: panic convertPropFromCache

```
I POST /v1/comments/new HTTP/1.1
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x9889fc]

goroutine 44 [running]:
main.convertPropFromCache(0xb68f89, 0x1, 0x4, 0x5ccca508, 0xc000c5db80, 0x40, 0xc000c5dc40, 0x40, 0xc000c88a80, 0x80,
 ...)
        /home/build/.go/src/github.com/decred/politeia/politeiawww/convert.go:289 +0x3fc
main.(*politeiawww).getProp(0xc0005b6000, 0xc000c5c4c0, 0x40, 0x0, 0x0, 0x0)
        /home/build/.go/src/github.com/decred/politeia/politeiawww/proposals.go:403 +0x1a7
main.(*politeiawww).getProposalAndAuthor(0xc0005b6000, 0xc000c5c4c0, 0x40, 0xc00006a660, 0x1000000010000, 0x1, 0x811e
04)
        /home/build/.go/src/github.com/decred/politeia/politeiawww/events.go:98 +0x5a
main.(*politeiawww)._setupCommentReplyEmailNotifications.func1(0xc0002e0600, 0xc0005b6000)
        /home/build/.go/src/github.com/decred/politeia/politeiawww/events.go:370 +0xc5
created by main.(*politeiawww)._setupCommentReplyEmailNotifications
        /home/build/.go/src/github.com/decred/politeia/politeiawww/events.go:361 +0x66

```

--- a/politeiawww/comments.go
+++ b/politeiawww/comments.go
@@ -388,7 +388,7 @@ func (p *politeiawww) processNewCommentInvoice(nc www.NewComment, u *user.User) 
 		return nil, err
 	}
 
-	// Add comment to commentScores in-memory ca
+	// Add comment to commentScores in-memory cache
 	p.Lock()
 	p.commentScores[nc.Token+ncr.CommentID] = 0
 	p.Unlock()

--- a/politeiawww/comments.go
+++ b/politeiawww/comments.go
@@ -388,7 +388,7 @@ func (p *politeiawww) processNewCommentInvoice(nc www.NewComment, u *user.User) 
 		return nil, err
 	}
 
-	// Add comment to commentScores in-memory cache
+	// Add comment to commentScores in-memory ca
 	p.Lock()
 	p.commentScores[nc.Token+ncr.CommentID] = 0
 	p.Unlock()
@@ -400,10 +400,13 @@ func (p *politeiawww) processNewCommentInvoice(nc www.NewComment, u *user.User) 
 	}
 
 	// Fire off new comment event
-	p.fireEvent(EventTypeComment, EventDataComment{
-		Comment: c,
-	})
-
+	/*
+		// XXX This is implemented only for proposal comments.  If we want email
+		// notifications for cms here is where to add the updated impls.
+		p.fireEvent(EventTypeComment, EventDataComment{
+			Comment: c,
+		})
+	*/
 	return &www.NewCommentReply{
 		Comment: *c,
 	}, nil

